<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vktracker</class>
 <widget class="QMainWindow" name="vktracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>vktracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 31, 32)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="tools">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>760</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(82, 129, 185);
border-style: solid;
border-width: 2px;
border-radius: 4px;
border-color: rgb(82, 129, 185)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-style: semi bold;
color: rgb(214, 215, 216);
background-color: rgb(81, 129, 184);
border-style: solid;</string>
     </property>
     <property name="text">
      <string>войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>140</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-style: semi bold;
color: rgb(214, 215, 216);
background-color: rgb(81, 129, 184);
border-style: solid;</string>
     </property>
     <property name="text">
      <string>загрузить список</string>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>15</y>
       <width>190</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../exvk/vktracker2.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="userPhoto">
     <property name="geometry">
      <rect>
       <x>522</x>
       <y>15</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>0</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-style: semi bold;
color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>вход не выполнен</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logout">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-style: semi bold;
color: rgb(214, 215, 216);
background-color: rgb(81, 129, 184);
border-style: solid;</string>
     </property>
     <property name="text">
      <string>выйти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cleanAll">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-style: semi bold;
color: rgb(214, 215, 216);
background-color: rgb(81, 129, 184);
border-style: solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>delete_all.png</normaloff>delete_all.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="enter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>760</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 26, 27);
border-style: solid;
border-width: 2px;
border-radius: 4px;
border-color: rgb(24, 26, 27);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>13</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 13px;
font-style: semi bold;
color:rgb(153, 188, 214)</string>
     </property>
     <property name="text">
      <string>Введите id пользователя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addLink">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>43</y>
       <width>720</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 4px;
border-color: rgb(70, 74, 77);
font-size: 15px;
font-style: semi bold;
color: rgb(214, 215, 216);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="lists" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>205</y>
      <width>760</width>
      <height>580</height>
     </rect>
    </property>
    <widget class="QListWidget" name="following">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>370</width>
       <height>580</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 26, 27);
border-style: solid;
border-width: 2px;
border-radius: 4px;
border-color: rgb(24, 26, 27);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="notification">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>0</y>
       <width>370</width>
       <height>580</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 26, 27);
border-style: solid;
border-width: 2px;
border-radius: 4px;
border-color: rgb(24, 26, 27);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
